python/Ganga/new_tests/GPI/Monitoring/TestMonitoring.py

```diff
@@ -23,10 +23,26 @@ def setUp(self):
 
     def tearDown(self):
         from Ganga.Utility.Config import getConfig
-        #getConfig('PollThread').getOption('autostart').revertToDefault()
-        #getConfig('PollThread').getOption('base_poll_rate').revertToDefault()
         super(TestMonitoring, self).tearDown()
 
+    def teat_0_testRegistryAsserts(self):
+        from Ganga.GPI import jobs, box, tasks, prep
+        from Ganga.GPIDev.Base.Proxy import stripProxy
+        from Ganga.Core.GangaRepository import getRegistryProxy, getRegistrySlice, getRegistry
+        assert getRegistryProxy('jobs') is jobs
+        assert getRegistryProxy('box') is box
+        assert getRegistryProxy('tasks') is tasks
+        assert getRegistryProxy('prep') is prep
+
+        assert getRegistrySlice('jobs') is stripProxy(jobs)
+        assert getRegistrySlice('box') is stripProxy(box)
+        assert getRegistrySlice('tasks') is stripProxy(tasks)
+        assert getRegistrySlice('prep') is stripProxy(prep)
+
+        assert getRegistry('jobs') is stripProxy(jobs).objects
+        assert getRegistry('tasks') is stripProxy(tasks).objects
+        assert getRegistry('box') is stripProxy(box).objects
```

Comment:
Maybe/possibly I'm not so sure at the moment. I'm debating this but the problem I was having with the Monitoring not working was that some of these asserts would fail. I can branch out into another test if needed it's only a small check.